.github/workflows/container.yml

```diff
@@ -52,6 +62,8 @@ jobs:
 
       - name: Push container
         if: ${{ github.event_name != 'pull_request' }}
+        env:
+          COSIGN_EXPERIMENTAL: true
         run: |
           make push-container
```

Comment:
We need a signed version of this, no?

Reply:
errr 🤦 forgot that, added, ptal
thanks so mush for your review